boardgamegeek/loaders/geeklist.py

```diff
@@ -0,0 +1,68 @@
+import datetime
+
+from ..utils import xml_subelement_text
+from ..objects.geeklist import GeekList
+
+
+def parse_date(str_date):
+    return datetime.datetime.strptime(str_date[:-6], '%a, %d %b %Y %H:%M:%S')  # Ignoring the timezone specifier. TODO: This is only valid as long as dates are provided in UTC time zones?
+    # example: Sat, 02 Feb 2019 15:13:54 +0000
+
+
+def add_geeklist_comments_from_xml(geeklist_or_item, xml_root):
+    added_comments = False
+    for comment in xml_root.findall("comment"):
+        # initial data for this collection item
+        data = {
+            "username": comment.attrib["username"],
+            "date": parse_date(comment.attrib["date"]) or None,
+            "postdate": parse_date(comment.attrib["postdate"]) or None,
+            "editdate": parse_date(comment.attrib["editdate"]) or None,
+            "thumbs": int(comment.attrib["thumbs"]),
+            "text": comment.text.strip()
+        }
+        listcomment = geeklist_or_item.add_comment(data)
+        added_comments = True
+    return added_comments
+
+
+def create_geeklist_from_xml(xml_root, listid):
+    data = {
+        'id': listid,
```

Comment:
code style consistency: double quoted strings

Reply:
Fixed in 5cd4b9e613defcae5f48fcd331037b9181ad6afc